src/app/authentication/authentication-routing.module.ts

```diff
@@ -1,10 +1,31 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { GuestGuard } from './guards/guest.guard';
+import { DefaultComponent } from './layouts/default/default.component';
 
-import { LoginComponent } from './pages';
+import { LoginComponent, ForgotPasswordComponent } from './pages';
 
 const routes: Routes = [
-  { path: 'login', component: LoginComponent }
+  {
+    path: 'auth',
+    component: DefaultComponent,
+    canActivate: [GuestGuard],
+    children: [
+      {
+        path: '',
```

Comment:
How about using wildcard routes so that it handles even the routes does match(ex: `/auth/invalid-path`)?
```suggestion
     ....
     ....
     {
        path: '**',
        redirectTo: 'login'
      },
```

Reply:
Thanks @junan, we migrate also add a 404 page later 💭